File: hackerrank.com/problem-solving/staircase.scala
// https://www.hackerrank.com/challenges/staircase/problem

import scala.io.StdIn._

object Solution {
  
  def staircase(n: Int): Unit = {
    for (i <- 1 to n) {
      println(("#" * i).padTo(n, ' ').reverse)
    }
  }
  
  def main(args: Array[String]) {

    val n = readInt
    staircase(n)
  }
}

File: hackerrank.com/problem-solving/staircase.js
// https://www.hackerrank.com/challenges/staircase/problem

function staircase(arr) {
  for (let i = 1; i <= n; i++) {
    console.log(`${'#'.repeat(i).padStart(n, ' ')}`)
  }
} 

function main(inputData) {
  const lines = inputData.split("\n");
  const N = parseInt(lines[0]);
  const arr = lines[1].split(' ').map(val => parseInt(val))
  staircase(arr)
}

process.stdin.resume();
process.stdin.setEncoding("ascii");
let _input = '';
process.stdin.on("data", input => {
  _input += input;
});

process.stdin.on("end", () => {
 main(_input);
});